### DISCUSSION TITLE
Activate ToDo mode in current active window/app with AppleScript or Shell?

### DISCUSSION THREAD

**Discussion by alexfazio** [rxhanson/Rectangle#758] [Ideas]
Is it possible to activate ToDo mode in current active window/app with AppleScript or Shell code?

I badly want a keyboard shortcut to activate todo mode on the active app, instead of having to click on the menubar icon.

**Comment by rxhanson** [comment]
Something that always gets pushed behind other items in priority. Feel free to beat me to it with a good PR.
